apps/namadillo/src/atoms/checksums/services.ts

```diff
@@ -7,11 +7,12 @@ export const fetchWasmHashes = async (
   integration: Namada
 ): Promise<WasmHash[]> => {
   const wasmHashes = await integration.getTxWasmHashes(chainId);
```

Comment:
Note: In another PR I'm removing these functions from the integration - this atom should only query for these when there is a new chain ID, and store locally, then provide as an (optional) argument to the extension when signing. As there is potential for a client to provide a bad set of wasm hashes, we shouldn't track it in the extension as it may break the Tx details! See https://github.com/anoma/namada-interface/issues/922